### DISCUSSION TITLE
In which SDK version will FSR be included?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#935] [Q&A] [answered]
Hi,
We need clarification regarding which SDK version FSR belongs to. It was communicated in March that it will be part of SDK 1.13. Can you confirm that or please inform us about the version where FSR will be introduced.

Kind regards,
Åsa

**Comment by YvesJo** [comment, marked as answer]
Hi,
For such questions, please get in touch with the helpdesk. As communicated during the March webinar, FSR will be in SDK 1.12, IPI will be part of SDK 1.13 (as the required implementing regulation is still missing).

**Comment by Colan71** [comment]
Hi,
It was actually communicated during the March webinar that FSR will be part of 1.13. 
![SDK](https://github.com/OP-TED/eForms-SDK/assets/79446518/0c1a996a-3252-46d8-b9c7-f647a544ab62)

**Reply by YvesJo** [reply]
Ok, so, at that time, it was still foreseen to have an interim SDK 1.12 version. SDK 1.12 & SDK 1.13 became SDK 1.12 and SDK 1.14 became 1.13

**Comment by ferrakaop** [comment]
This is a good question as the SDK roadmap has changed compared to what we said in March.  We emailed eSenders yesterday (to Developer Portal profiles in Preview and Production) and updated the roadmap: https://docs.ted.europa.eu/eforms-common/roadmap/

In a nutshell: SDK 1.12 is now for June with business entities, FSR and exclusion/selection criteria, SDK 1.13 is the big one in October with IPI, EED and the 6 voluntary forms.
